specification/search/data-plane/Azure.Search/preview/2020-06-30-Preview/searchservice.json

```
@@ -7324,7 +7324,7 @@
         },
         "entitiesDefinitionUri": {
           "type": "string",
-          "description": "Path to a JSON or CSV file containing all the target text to match against."
+          "description": "Path to a JSON or CSV file containing all the target text to match against. This entity definition is read at the beginning of an indexer run; any updates to this file mid-run won't be realized until subsequent runs. This config must be accessible over HTTPS."
```

I recommend a more translator-friendly version:
```suggestion
          "description": "Path to a JSON or CSV file containing all the target text to match against. This entity definition is read at the beginning of an indexer run. Any updates to this file during an indexer run will not take effect until subsequent runs. This config must be accessible over HTTPS."
```